ChineseFilter/sections/general_extensions.txt

```diff
@@ -66,6 +66,7 @@ v-wb.youku.com#%#//scriptlet('set-constant', 'Object.prototype.adData', 'emptyOb
 ! https://github.com/AdguardTeam/AdguardFilters/issues/26318
 ! https://forum.adguard.com/index.php?threads/youku-com-v-qq-com.30749/
 youku.com#%#//scriptlet('adjust-setTimeout', 'e-', '', '0.02')
+youku.com#%#//scriptlet("adjust-setTimeout", "r--", "", "0.02")
```

Comment:
Could you replace the quotes to ordinary ones? Like this
```adblock
youku.com#%#//scriptlet('adjust-setTimeout', 'r--', '', '0.02')
```

Reply:
Please just a sec.

Reply:
In fact, it's not necessary, but to follow the same order.